src/Elmish.WPF/ViewModel.fs

```diff
@@ -502,8 +489,11 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
     | SubModelSelectedItem _ ->
         None
     | Cmd (cmd, canExec) ->
-        if canExec newModel = canExec currentModel then None else Some cmd
-    | CmdParam cmd -> Some cmd
+        if canExec newModel = canExec currentModel
```

Comment:
I can accept this, but in general I see nothing wrong having a short if-then-else on a single line.

Reply:
I agree.  I found it easier to read at the time.  Since you will accept this, I will leave it as is.